src/seedcase_sprout/core/path_local.py

```diff
@@ -0,0 +1,290 @@
+    """
+    path = path_resource(resource_id, path=path) / "data.parquet"
+    return check_is_file(path)
+
+
+def path_resource_batch(resource_id: int, path: Path = Path.cwd()) -> Path:
+    """Gets the absolute path to a specific resource's batch folder.
+
+    Args:
+        resource_id: The ID of the resource.
+        path: Provide a path to the package directory. Defaults to the current working
+            directory.
+
+    Returns:
+        The absolute path to a specific resource's batch folder in a data package.
+
+    Examples:
+        ```{python}
+        import tempfile
+        from pathlib import Path
+
+        import seedcase_sprout.core as sp
+
+        # Create a temporary directory for the example
+        with tempfile.TemporaryDirectory() as temp_dir:
+            temp_path = Path(temp_dir)
+
+            # Create a package and resource structure first
+            sp.create_package_properties(
+                properties=sp.example_package_properties(),
+                path=temp_path
+            )
+
+            resources_path = Path(temp_path / "resources")
+            resources_path.mkdir()
+            sp.create_resource_structure(path=resources_path)
+
+            # Get the path to the resource's batch folder
+            sp.path_resource_batch(resource_id=1, path=temp_path)
+        ```
+    """
+    path = path_resource(resource_id, path=path) / "batch"
+    return check_is_dir(path)
+
+
+def path_resource_batch_files(resource_id: int, path: Path = Path.cwd()) -> list[Path]:
+    """Gets the absolute path to the batch files of a specific resource.
+
+    Args:
+        resource_id: The ID of the resource.
+        path: Provide a path to the package directory. Defaults to the current working
+            directory.
+
+    Returns:
+        A list of paths to a specific resource's batch files in a data package.
+
+    Examples:
+        ```{python}
+        import tempfile
+        from pathlib import Path
+
+        import seedcase_sprout.core as sp
+
+        # Create a temporary directory for the example
+        with tempfile.TemporaryDirectory() as temp_dir:
+            temp_path = Path(temp_dir)
+
+            # Create a package and resource structure first
+            sp.create_package_properties(
+                properties=sp.example_package_properties(),
+                path=temp_path
+            )
+
+            resources_path = Path(temp_path / "resources")
+            resources_path.mkdir()
+            sp.create_resource_structure(path=resources_path)
+            # TODO: Add data/batch files to resource
+            # sp.write_resource_batch_data(
+            #     path=sp.path_resource_batch(resource_id=1, path=temp_dir),
+            #     data="path/to/data.csv")
+
+            # Get the path to the resource's batch files
+            # sp.path_resource_batch_files(resource_id=1, path=temp_dir)
+        ```
+    """
+    return list(path_resource_raw(resource_id, path=path).iterdir())
```

Comment:
```suggestion
    return list(path_resource_batch(resource_id, path=path).iterdir())
```
Right?
Also other references to `path_resource_raw` and `path_resource_raw_files` in the init file and the tests